internal/cli/apps.go

```diff
@@ -479,7 +479,7 @@ func createAppCmd(cli *cli) *cobra.Command {
 				return fmt.Errorf("Unable to create application: %v", err)
 			}
 
-			if err := cli.setDefaultAppID(a.GetClientID()); err != nil {
+			if err := cli.Config.SaveNewDefaultAppIDForTenant(cli.tenant, a.GetClientID()); err != nil {
```

Comment:
Maybe `SaveDefaultAppIDForTenant()`. Seems overly verbose. Totally a nit, though. Not a blocker.

Reply:
Yeah I like that to be honest, I'll do the same with SetDefaultTenant. 👍🏻 Thanks @Widcket

Reply:
[`954f34b` (#743)](https://github.com/auth0/auth0-cli/pull/743/commits/954f34b6a9e6a6cb4854c476e10f6a6d6c054f1d) I also realized we can set the path to private instead of having it public.